Sources/LanguageServerProtocol/Messages.swift

```diff
@@ -36,6 +36,7 @@ public let builtinRequests: [_RequestType.Type] = [
   DocumentColorRequest.self,
   DocumentDiagnosticsRequest.self,
   DocumentFormattingRequest.self,
+  DocumentRangeFormattingRequest.self,
```

Comment:
`DocumentRangeFormattingRequest` is already in this array and that’s causing crashes during test execution and thus CI to fail

Reply:
👌